nvflare/app_common/storages/filesystem_storage.py

```diff
@@ -26,18 +27,20 @@
 
 
 def _write(path: str, content):
+    tmp_name = "_" + str(uuid.uuid4())
```

Comment:
Note: a minor improvement could be have just
`tmp_name = path + "_" + str(uuit.uuid4())`

and use tmp_name afterwards